Frends.AzureDataLake.ListFiles/README.md

```diff
@@ -0,0 +1,34 @@
+# Frends.AzureDataLake.ListFiles
```

Comment:
Just in general - please make sure that solution building does not produce any warnings.